make.config.in

```diff
@@ -309,6 +309,7 @@ clean::
 	-@$(RM) -rf $(OBJ) $(DEPS) $(TARGET)
 	@for pp in $(DIRS); do echo "  " $$pp cleaned; $(MAKE) --no-print-directory -C $$pp clean; done
 	@$(RM) -f $(SUB_LIBS)
+	@$(RM) .*.mk
```

Comment:
I couldn't find that we set $(RM) to anything. It seems to be `rm -f` - but in case it is only `rm` this could/would fail ...

Reply:
`$(RM)` is an implicit `make` variable which defaults to `rm -f`: https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html